go/libkb/api.go

```diff
@@ -322,11 +322,15 @@ var lastUpgradeWarning *time.Time
 func (a *InternalAPIEngine) consumeHeaders(resp *http.Response) error {
 	u := resp.Header.Get("X-Keybase-Client-Upgrade-To")
 	p := resp.Header.Get("X-Keybase-Upgrade-URI")
+	m := resp.Header.Get("X-Keybase-Upgrade-Message")
```

Comment:
We're probably going to want the option of putting newlines or other formatting in this message. That doesn't interact well with HTTP headers, so we'll want to assume some kind of encoding from the server, and do the corresponding decoding here. Is base64 safe?